postable.rb

```diff
@@ -0,0 +1,16 @@
+require 'uri'
```

Comment:
Looks like we can remove the entire postable.rb file.

Reply:
Removed postable.rb